pixy/httpapi.go

```diff
@@ -0,0 +1,131 @@
+package pixy
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/gorilla/mux"
+	"github.com/mailgun/kafka-pixy/Godeps/_workspace/src/github.com/mailgun/manners"
+)
+
+const (
+	NetworkTCP  = "tcp"
+	NetworkUnix = "unix"
+
+	hContentLength = "Content-Length"
+
+	fTopicName  = "topic"
+	fRoutingKey = "key"
```

Comment:
The meanings of `f` and `h` are not obvious to me. I think I would just call these `contentLength`, `topicName`, and `routingKey`. Or `contentLength`, `topic`, and `key`.